docs/releases/1.10.txt

```diff
@@ -138,7 +147,10 @@ File Uploads
 Forms
 ^^^^^
 
-* ...
+* Widget media gets now served from using ``staticfiles`` if
+  configured.
+* :meth:`django.forms.widgets.Media.absolute_path` removed
```

Comment:
No I searched on github, I could only fine one small repo that does monkey patching there.